eth-balance/src/lib.rs

```diff
@@ -21,13 +24,34 @@ fn store_balance(block: pbeth::v2::Block, output: store::StoreSetRaw) {
                             .into()
                     })
                     .unwrap_or(BigInt::zero());
-
-                output.set(
-                    transaction.end_ordinal,
-                    format!("Address:{}", Hex(&balance_change.address).to_string()),
-                    &new_value.to_string(),
-                )
+                let new_token_balance = vec![token::TokenBalance {
+                    token: Some(get_eth_token()),
+                    balance: new_value.to_string(),
+                }];
+                let account = token::Account {
+                    address: Hex(&balance_change.address).to_string(),
+                    balances: new_token_balance,
+                };
+                accounts.push(account);
             }
         }
     }
+
+    Ok(token::Accounts { items: accounts })
+}
+
+//////////////////////////
+//// Helper Functions ////
+//////////////////////////
+
+fn get_eth_token() -> token::Token {
```

Comment:
should move this to `substreams-helper`

Reply:
Moved, but still struggling to match the type